src/controller/kobuki_controller.cpp

```diff
@@ -170,6 +191,7 @@ void KobukiController::moveWithSTC() {
   // 500ms
   ros::Rate loop_rate(2);
   loop_rate.sleep();
+  for(int k=0;k<500000000;k++){}
```

Comment:
không cần vòng lặp này đâu

Reply:
sau này khi sử dụng đến amcl, t muốn cho currentCellX, currentCellY trong hàm amcl được cập nhật mấy trăm lần cho nó chính xác, nên t cho vòng for